### ISSUE TITLE
Feature request: add amend option for editing

### ISSUE THREAD

**Issue by goldhoorn** [isaacs/github#439]
When modifying a PR, there are currently two option
" Commit directly to the xxx branch" 
and
" Create a new branch for this commit and start a pull request. Learn more about pull requests."

A new option "amend the changed to last commit (require force-push rights)" would be a great option. 

Use case: trivial PR that consist only of one commit that contain a typo are easy to change. Often the developer requests a squeeze of this commit. The local way is for such changes a bit hard. The option to do this online on the web-gui would be really great!

**Comment by cirosantilli** [comment]
GitLab declined this :-( http://feedback.gitlab.com/forums/176466-general/suggestions/5545069-commit-amend-from-web-interface

In case you want to correct an error on a separate feature branch, squash https://github.com/isaacs/github/issues/406 would be an option.

**Comment by goldhoorn** [comment]
Uhh that's bad news i would still love this. even force-pushing on this branches is the normal workflow and it's done

**Comment by cirosantilli** [comment]
@goldhoorn yes, totally disagree with GL there. They've even implemented rebase from web UI http://feedback.gitlab.com/forums/176466-general/suggestions/4289653-rebase-merge-requests-in-the-web-ui which is a force push...
